src/etc/init.d/S50dropbear

```diff
@@ -13,25 +13,33 @@ start() {
 
 	echo -n "Starting dropbear sshd: "
 	umask 077
-	# Ensure host keys are changed when instance ID changes
-	cirros-per instance remove-dropbear-host-keys -- rm -rf /etc/dropbear
-	
-	# Make sure dropbear directory exists
-	if [ ! -d /etc/dropbear ]; then
-		mkdir -p /etc/dropbear
+	local dropbear_key_dir="/etc/dropbear"
+
+	# Handle symlinked directories
+	if [ -L "$dropbear_key_dir" ]
+	then
+		dropbear_key_dir=$(readlink "$dropbear_key_dir")
+	fi
+
+	if [ -d "$dropbear_key_dir" ]
+	then
+		# Ensure host keys are changed when instance ID changes
+		cirros-per instance remove-dropbear-host-keys -- rm -rf $(dropbear_key_dir)
```

Comment:
I think this is a bug. you're going to *invoke* `$dropbear_key_dir`.

You want:

    rm -rf "${dropbear_key_dir}"

Reply:
Ouch... right. This worked for me probably because it was never called... I think plain `"$dropbear_key_dir"` will do fine too, right?

Reply:
yes.